tests/queries/0_stateless/01515_logtrace_function.sh

```diff
@@ -0,0 +1,6 @@
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../shell_config.sh
+
+CLICKHOUSE_CLIENT=$(echo ${CLICKHOUSE_CLIENT} | sed 's/'"--send_logs_level=${CLICKHOUSE_CLIENT_SERVER_LOGS_LEVEL}"'/--send_logs_level=debug/g')
```

Comment:
Ok. Also you can just set CLICKHOUSE_CLIENT_SERVER_LOGS_LEVEL before loading  shell_config.